src/core/InstrumentFunctions.cpp

```diff
@@ -136,7 +136,7 @@ InstrumentFunctionNoteStacking::ChordTable::Init InstrumentFunctionNoteStacking:
 	{ QT_TRANSLATE_NOOP( "InstrumentFunctionNoteStacking", "Half-Whole Diminished" ), { 0, 1, 3, 4, 6, 7, 9, 10, -1 } },
 	
 	{ QT_TRANSLATE_NOOP( "InstrumentFunctionNoteStacking", "5" ), { 0, 7, -1 } },
-	{ QT_TRANSLATE_NOOP( "InstrumentFunctionNoteStacking", "Persian" ), { 0, 1, 4, 5, 7, 8, 10, -1 } }
+	{ QT_TRANSLATE_NOOP( "InstrumentFunctionNoteStacking", "Persian" ), { 0, 1, 4, 5, 6, 8, 11, -1 } }
 } ;
```

Comment:
I liked this one too: { 0, 1, 4, 5, 7, 8, 10, -1 } and tracked it down.
https://en.wikipedia.org/wiki/Phrygian_dominant_scale
I suggest adding it too as **Phrygian dominant** or **Phrygian major** .